pkg/mail/server.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information
+
+package mail
+
+import (
+	"crypto/tls"
+	"net"
+	"net/smtp"
+
+	"github.com/zeebo/errs"
+)
+
+// SMTPServer is smtp server
+type SMTPServer struct {
```

Comment:
Maybe `Client`/`Mailer`/`Proxy`/`Sender`, because it isn't actually a `Server` that listens for incoming messages.